.github/workflows/wan-perf.yml

```diff
@@ -0,0 +1,110 @@
+name: WAN Perf
+
+on:
+  push:
+    branches:
+    - main
+    - release/*
+    paths-ignore:
+    - .azure/*
+    - docs/*
+    - src/cs/*
+    - src/distribution/*
+    - src/plugins/*
+    - src/tools/*
+    - src/lib.rs
+    - Cargo.toml
+    - README.md
+  pull_request:
+    branches:
+    - main
+    - release/*
+    paths-ignore:
+    - .azure/*
+    - docs/*
+    - src/cs/*
+    - src/distribution/*
+    - src/plugins/*
+    - src/tools/*
+    - src/lib.rs
+    - Cargo.toml
+    - README.md
+
+env:
+  seed: 41473a2e60b6958500ec0add7dcfb9 # TODO - Randomize?
```

Comment:
File an issue to randomize this so we don't lose track of it? WAN perf won't vary ever if this is always hard-coded.